### ISSUE TITLE
Details of Training with Randomized Channel Shuffling in Section 3.3

### ISSUE THREAD

**Issue by Raytsang123** [VITA-Group/Random-Shuffling-BackdoorDetect#1]
Could you provide the code for generating the shuffled variant with low clean accuracy and high ASR ?

In the paper, authors mentioned that a few training epochs (5 epochs) on a subset (10%) of the original training set are sufficient to generate a high quailty shuffled variant. Does the subset (10%) contain poisoned samples ?

Why does randomized channel shuffling in every iteration stop the model to encode meaningful clean image features ? The rationale of Section 3.3 seems to be contradictary with the observation in Section 3.2. In Section 3.2, authors point out that benign features are more robust to channel shuffling while backdoor features are less robust to channel shuffling.

Thank you a lot if you could elaborate more on the details of training with Randomized Channel Shuffling.
